### DISCUSSION TITLE
Getting Started Errors

### DISCUSSION THREAD

**Discussion by HoboDiem** [Chia-Network/chia-blockchain#702] [Q&A] [answered]
Hi All, I'm on Windows 10 with fresh install of everything. I've created my first test plot and am receiving this error. Going through Wiki and the Github I'm pretty confident I missed a step somewhere.  I'm likely one of the lowest technically skilled here, so maybe my question is more representative of how a little extra language in the Windows set up language could broaden your farming/user base.  Thanks in advance.
![Plotting Error](https://user-images.githubusercontent.com/77577815/104845976-33b13900-58a6-11eb-9fdc-d4fef4dc63b6.PNG)

**Comment by hoffmang9** [comment, marked as answer]
The windows user you're logged in as apparently doesn't have the ability to create files in the root directory of C:\. Just make a directory and then you will have the ability to write to it.

**Reply by HoboDiem** [reply]
Resolved. I hadn't given Chia Admin rights. All sorted. Thanks!

**Comment by xxxxxxxWWQ43Q4QEG2TG** [comment]
hi im have question  the chia newtork not say is a prbole but then not goin more the 11 or 12% are know what is goin on

**Reply by xxxxxxxWWQ43Q4QEG2TG** [reply]
the chia not goin more the 11 or 12% but dont say is a bug just not goin more the that
